plugins/email_service/app/controllers/email_service/web_controller.rb

```diff
@@ -0,0 +1,10 @@
+module EmailService
+  class WebController < ::EmailService::ApplicationController
```

Comment:
How you enforce your policy? I couldn't find any reference to the 
```
authorization_context 'email_service'
authorization_required
```

Reply:
done.